javascript/src/main/components/Nav/AppNavbar.js

```diff
@@ -42,9 +42,15 @@ function AppNavbar() {
               </NavDropdown>
           }
         <LinkContainer to={"/about"}>
-            <Nav.Link>About</Nav.Link>
+          <Nav.Link>About</Nav.Link>
         </LinkContainer>
         <ProfileNav />
+        <NavDropdown title="Analyze Message Data">
```

Comment:
This menu item should only appear if the user is logged in as an admin or member.

Look at the code earlier in the file for an example, for instance:
```
      { (isMember || isAdmin)  &&
              ...
```